clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/services/UsersMessageReceiver.as

```diff
@@ -172,16 +173,14 @@ package org.bigbluebutton.lib.user.services {
 			userSession.joinMeetingResponse(msg);
 		}
 		
-		private function handleRecordingStatusChanged(m:Object):void {
-			var msg:Object = JSON.parse(m.msg);
+		private function handleRecordingStatusChanged(msg:Object):void {
```

Comment:
All of the message handler function have the name "handle<MessageName>". Can you please change this to match.